New branch with link tests

Slangman3 opened the pull request:
Check please.

Slangman3 committed:
Nothing was added, just clean code from not needed strings.

Slangman3 committed:
Added for pull request test about links.

dzmitryrak commented on src/test/java/ChatLinkTest.java:
```diff
@@ -0,0 +1,34 @@
+        btnLinkWithCode.click();
+        Clipboard c2 = Toolkit.getDefaultToolkit().getSystemClipboard();
+        assertNotNull(c2);
+        System.out.println(c2.getData(DataFlavor.stringFlavor));
```
никаких sout

dzmitryrak commented on src/test/java/ChatLinkTest.java:
```diff
@@ -0,0 +1,34 @@
+        WebElement btnLinkWithCode = driver.findElement(By.xpath("//*[@id='invite-users-to-chat']"));
+        btnLinkWithCode.click();
+        Clipboard c2 = Toolkit.getDefaultToolkit().getSystemClipboard();
+        assertNotNull(c2);
```
проверка на NUll лишняя

dzmitryrak commented on src/test/java/ChatLinkTest.java:
```diff
@@ -0,0 +1,34 @@
+        ChromeDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        driver.get("https://dev.integrivideo.com/");
+        WebElement startChatBtn = driver.findElementByXPath("/html/body/main/div/div[2]/div/div[3]/div[2]/div/div/a");
```
ужасный xpath

Slangman3 committed:
Added input enter test.

Slangman3 committed:
Added POM chat page on test.

Slangman3 committed:
Added POM chat settings on test.

Slangman3 committed:
Added chat tests.

Slangman3 committed:
Added chat's settings tests.

Slangman3 commented:
Запушил новые файлы, а новую брнчу не создавал, по итогу только здесь видны в плохом виде мои пуши.

Slangman3 committed:
Added to POM chat new selectors and methods.

Slangman3 committed:
Added doc with 1000 simbols to check list value.

Slangman3 committed:
Added some tests.
Отправить любое сообщение используя Enter
Отправить любое сообщение используя Кнопку
Отправить сообщение из 1000 символов
Отправить сообщение со ссылкой
Отправить сообщение содержащее Javascript code
Отредактировать сообщение
Изменить имя
Изменить email
Изменить Photo URL
Проверить работу пноки Invite
Проверить работу кнопки с кодом

Slangman3 committed:
Added tests about edit and remove text and message.

Slangman3 committed:
Added tests about sending 11+ messages and drug and drop option.

dzmitryrak commented on src/test/java/tests/ChatInputIsEmptyTest.java:
```diff
@@ -0,0 +1,163 @@
+    }
+
+    @Test
+    public void inputEnter() throws InterruptedException {
```
лишние 2 теста, были в другом классе вроде
